Formula/n/notmuch.rb

```diff
@@ -57,6 +58,8 @@ def install
                             "--without-ruby"
       system "make", "V=1", "install"
     end
+    chmod 0755, "notmuch-git"
+    bin.install "notmuch-git"
```

Comment:
I dont think you need to update file permission as there is already `chmod ugo+x` in the Makefile?

But you do need to update the shebang (you need to also include `  include Language::Python::Shebang` in L2)

```suggestion
    bin.install "notmuch-git"
    rw_info = python_shebang_rewrite_info("#{libexec}/bin/python")
    rewrite_shebang rw_info, bin/"notmuch-git"
```

Reply:
I couldn't get the suggestion for rewriting the shebang to work, so I followed the example given in the brew formula cookbook. Otherwise makes sense. Audit and test pass for me.

(sorry for the double push.)

Reply:
that also works, thanks!